Formula/carla.rb

```diff
@@ -0,0 +1,27 @@
+class Carla < Formula
+  desc "Audio plugin host supporting LADSPA, LV2, VST2/3, SF2 and more"
+  homepage "https://kxstudio.linuxaudio.org/Applications:Carla"
+  url "https://github.com/falkTX/carla", :using => :git, :tag => "v1.9.9", :revision => "c03571a9ef95ac0e9564b95347f5de819aa7fb54"
+  revision 0
+
+  depends_on "pkg-config" => :build
+  depends_on "fluid-synth"
+  depends_on "liblo"
+  depends_on "libmagic"
+  depends_on :macos => :mavericks
+  depends_on "pyqt"
+  depends_on "python"
+
+  def install
+    args = %W[
+      PREFIX=#{prefix}
+    ]
+
+    system "make"
+    system "make", "install", *args
+  end
+
+  test do
+    system bin/"carla", "--version"
```

Comment:
Any chance of a test that exercises a deeper level of functionality? Thanks!

Reply:
The library author confirms `carla --version` is adequate (it uses `pyqt`, so this will show errors if compilation failed).  He's provided me a subsequent test... `system bin/"carla-discovery-native", internal, :all`.  It will be tested locally and added to this PR.